dsp-api-main/src/main/scala/dsp/api/main/DspMain.scala

```diff
@@ -0,0 +1,41 @@
+package dsp.api.main
+
+import zio._
+import dsp.user.route.UserRoutes
+import dsp.user.handler.UserHandler
+import dsp.user.repo.impl.UserRepoLive
+import zio.logging.removeDefaultLoggers
+import zio.metrics.connectors.MetricsConfig
+import zio.logging.backend.SLF4J
+import dsp.config.AppConfig
+import dsp.util.UuidGeneratorLive
+
+object DspMain extends ZIOAppDefault {
+
+  /**
+   * Configures Metrics to be run at a set interval, in this case every 5 seconds
+   */
+  val metricsConfig = ZLayer.succeed(MetricsConfig(5.seconds))
```

Comment:
Currently nowhere, it is referenced on line 37 but commented out. I thought it might be useful in the future. But you are right, it is confusing and I will just delete this.